lesson10/script.js

```diff
@@ -9,7 +9,7 @@ function removeLoadingImg(){
     document.getElementById('js-loading').remove();
 }
 
-function getDataAfter3Seconds(){
+function getData(){
     return new Promise((resolve,reject) => {
         const categriesData = [
             {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"},
```

Comment:
I rename the function to sen-san's idea,
because it is more easy to understand.